src/components/shared/AddRemoveForm.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react'
 import Buttons from '../buttons/index'
-import Settings from '../settings/index'
-import utils from '../../utils'
+// import Settings from '../settings/index'
```

Comment:
We haven't mocked out our `Settings` component yet, so tests will fail because `Settings` currently relies on `window.localStorage`. We also need to move it to the `src/` folder since it is not a component but a helper. These can be done together in another refactor.